app/Models/User.php

```diff
@@ -41,4 +41,5 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+    protected $guarded = array('is_admin');
```

Comment:
```suggestion
    protected $guarded = ['is_admin'];
```

Từ php 7.0 trở đi dùng 
```php
$a = [];
// để khai báo mảng thay vì
$a = array(); //nhe
```